pkg/plugins/hybrid/v1alpha/scaffolds/internal/templates/makefile.go

```diff
@@ -111,8 +111,11 @@ help: ## Display this help.
 
 ##@ Build
 
+build: generate fmt vet ## Build manager binary.
+	go build -o bin/manager main.go
+
 run: helm-operator ## Run against the configured Kubernetes cluster in ~/.kube/config
-	$(HELM_OPERATOR) run
+	go run ./main.go
```

Comment:
I believe we would need to provide a `watches.yaml` file path as a flag when running locally otherwise which we would error in [here](https://github.com/operator-framework/helm-operator-plugins/blob/6e30bde2868879811b3bedb200e911ae440413c8/testdata/hybrid/memcached-operator/main.go#L112).